vendor/vendor.json

```diff
@@ -168,10 +168,10 @@
 			"versionExact": "v2.0.0-preview.5"
 		},
 		{
-			"checksumSHA1": "1AyIehd0sjJ9uJoLZyqq9L9a1Jg=",
+			"checksumSHA1": "427ei4bQ6ap8N/Hrw7NpxOqjd+Y=",
```

Comment:
I wonder if it would be worth doing a separate PR to just update to this dependency, making sure it has not side effects. And then add the two additional packages in this PR.

Reply:
I don't mind that. But I'm indifferent. Shall I do that?

Reply:
OK, having tried to fix the deps, I'll definitely do that tomorrow. Looks like the latest libs don't make lambdabeat happy.

Reply:
👍